Buildings/Experimental/DHC/Networks/Controls/AgentPump1Pipe.mo

```diff
@@ -0,0 +1,238 @@
+within Buildings.Experimental.DHC.Networks.Controls;
+block AgentPump1Pipe
+  "Ambient network storage and plants agent pump control, developed for reservoir network"
+   parameter Buildings.Controls.OBC.CDL.Types.SimpleController controllerType=
+      Buildings.Controls.OBC.CDL.Types.SimpleController.P "Type of controller";
+   parameter Real yPumMin(min=0, max=1, final unit="1") = 0.05
+    "Minimum pump speed";
+   parameter Real yPumMax(min=0, max=1, final unit="1") = 1 "Upper limit of output";
+   parameter Real k = 1 "Gain of controller";
+   parameter Real Ti = 0.5 "Time constant of integrator block";
+   parameter Real Td = 0.1 "Time constant of derivative block";
+   parameter Modelica.Units.SI.TemperatureDifference dToff(min=0.1) = 1
+    "Temperature offset to account for heat exchanger pinch point";
+   parameter Real uLowHea = 1 "if y=true and u<uLow, switch to y=false";
+   parameter Real uHighHea = 2 "if y=false and u>uHigh, switch to y=true";
+   parameter Real uLowCoo = 1 "if y=true and u<uLow, switch to y=false";
+   parameter Real uHighCoo = 2 "if y=false and u>uHigh, switch to y=true";
+   parameter Real h = 0.15 "Hysteresis for net demand temperature calculation";
+
+  Buildings.Controls.OBC.CDL.Interfaces.RealInput TSouIn(final unit="K",
+       displayUnit="degC") "Temperatures at the inlet of the source"
+    annotation (Placement(transformation(extent={{-160,50},{-120,90}})));
+  Buildings.Controls.OBC.CDL.Interfaces.RealInput TSouOut(final unit="K",
+      displayUnit="degC") "Agent supply temperature"
+    annotation (Placement(transformation(extent={{-160,-30},{-120,10}})));
+  Buildings.Controls.OBC.CDL.Interfaces.RealInput TSou(final unit="K",
+      displayUnit="degC") "Average temperature available at source"
+    annotation (Placement(transformation(extent={{-160,10},{-120,50}})));
+  Buildings.Controls.OBC.CDL.Interfaces.RealInput TretDis(final unit="K",
+      displayUnit="degC") "District return temperature"
+    annotation (Placement(transformation(extent={{-160,-70},{-120,-30}})));
+  Buildings.Controls.OBC.CDL.Interfaces.RealInput TsupDis(final unit="K",
```

Comment:
This should be `TRetDis` and `TSupDis`.

Reply:
Thank you for catching the typos I updated the names in model and documentation